package.json

```diff
@@ -39,6 +39,6 @@
     "typescript": "^3.8.3"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=10"
```

Comment:
I don't see any problem here... but I don't remember all the details... cc @jdalton probably have some thoughts here... I think it was because we didn't want to compile... instead just use the thing directly in browsers using modules.

Reply:
The compiler can be Node 14 to avoid compilation though if we introduce it anyways as a way to ship it then we can just lower what is required to whatever the base Node we want to support is. In both secure-javascript-environment and the compiler we should use babel and the env-preset to specify the min Node version supported so it will know which things to transpile.

Reply:
I think so too, but what problem are we solving by downgrading this?